packages/openid4vc-client/src/OpenId4VcClientService.ts

```diff
@@ -54,7 +49,7 @@ export type RequestCredentialOptions = { flowType: AuthFlowType } & PreAuthCodeF
 // because we assume it will always be SHA256
 // as clear text code_challenges are unsafe
 export interface GenerateAuthorizationUrlOptions {
-  initiationUri: string
+  openidCredentialOffer: string
```

Comment:
Not too sure on this name. I did not want to call it `credentialOfferUri` as that is also a property inside the credential offer parameters. I think that would cause confusion, but I don't think that this name is the best we can do.

Reply:
What does the sphereon library use?

Reply:
Just `uri` which I think it not very descriptive. But it might be fine as I think it is done for backwards compatibility with 8/9 which uses initiateUri instead of credentialOfferUri.

Reply:
That is indeed the reason for it

Reply:
Thank you for the confirmation @nklomp. I will update the field to be `uri` as well.

Reply:
Also the context of the object it is a property of should be helpful enough to be clear about what the URI is about